[2025-07-26T09:01:27.956+0700] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-07-26T09:01:28.668+0700] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_local_postgres_pipeline.load_time_dim_table scheduled__2025-07-26T02:00:00+00:00 [queued]>
[2025-07-26T09:01:28.854+0700] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_local_postgres_pipeline.load_time_dim_table scheduled__2025-07-26T02:00:00+00:00 [queued]>
[2025-07-26T09:01:28.855+0700] {taskinstance.py:2306} INFO - Starting attempt 1 of 4
[2025-07-26T09:01:29.047+0700] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): load_time_dim_table> on 2025-07-26 02:00:00+00:00
[2025-07-26T09:01:29.056+0700] {logging_mixin.py:188} WARNING - /home/ngtph/airflow/airflow_venv/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=6609) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-07-26T09:01:29.057+0700] {standard_task_runner.py:63} INFO - Started process 6618 to run task
[2025-07-26T09:01:29.057+0700] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'etl_local_postgres_pipeline', 'load_time_dim_table', 'scheduled__2025-07-26T02:00:00+00:00', '--job-id', '541', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline_pjt.py', '--cfg-path', '/tmp/tmphgh26m_u']
[2025-07-26T09:01:29.058+0700] {standard_task_runner.py:91} INFO - Job 541: Subtask load_time_dim_table
[2025-07-26T09:01:32.249+0700] {task_command.py:426} INFO - Running <TaskInstance: etl_local_postgres_pipeline.load_time_dim_table scheduled__2025-07-26T02:00:00+00:00 [running]> on host DESKTOP-9M5T9P5.localdomain
[2025-07-26T09:01:36.709+0700] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='phuongnt226' AIRFLOW_CTX_DAG_ID='etl_local_postgres_pipeline' AIRFLOW_CTX_TASK_ID='load_time_dim_table' AIRFLOW_CTX_EXECUTION_DATE='2025-07-26T02:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-07-26T02:00:00+00:00'
[2025-07-26T09:01:36.710+0700] {taskinstance.py:430} INFO - ::endgroup::
[2025-07-26T09:01:37.632+0700] {base.py:84} INFO - Using connection ID 'postgres_id' for task execution.
[2025-07-26T09:01:38.244+0700] {sql.py:470} INFO - Running statement: 
    INSERT INTO time (start_time, hour, day, week, month, year, weekday)
    SELECT 
        start_time, 
        EXTRACT(hour FROM start_time), 
        EXTRACT(day FROM start_time), 
        EXTRACT(week FROM start_time), 
        EXTRACT(month FROM start_time), 
        EXTRACT(year FROM start_time), 
        EXTRACT(dow FROM start_time)
    FROM songplays
    ON CONFLICT (start_time) DO NOTHING                     
, parameters: None
[2025-07-26T09:01:38.757+0700] {sql.py:479} INFO - Rows affected: 0
[2025-07-26T09:01:38.802+0700] {logging_mixin.py:188} INFO - Executed SQL: 
    INSERT INTO time (start_time, hour, day, week... -> Result: None
[2025-07-26T09:01:38.803+0700] {python.py:237} INFO - Done. Returned value was: None
[2025-07-26T09:01:38.803+0700] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-07-26T09:01:39.534+0700] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=etl_local_postgres_pipeline, task_id=load_time_dim_table, run_id=scheduled__2025-07-26T02:00:00+00:00, execution_date=20250726T020000, start_date=20250726T020128, end_date=20250726T020139
[2025-07-26T09:01:39.803+0700] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-07-26T09:01:42.716+0700] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-07-26T09:01:42.823+0700] {local_task_job_runner.py:222} INFO - ::endgroup::
